src/components/Label/Label.scss

```diff
@@ -250,5 +204,14 @@ $transitionTimingFunction: ease-in-out;
                 var(--yc-color-text-complementary)
             );
         }
+
+        &_clear {
+            @include themeState(
+                none,
```

Comment:
`none` is not valid for `background-color`. Either use `background` property or change `none` to `transparent`